### ISSUE TITLE
Discrepancy in accessModeSufficient Metadata Requirement

### ISSUE THREAD

**Issue by Iacobus1983** [daisy/ace#400]
- Ace version: 1.3.2 (GUI)
- My OS: MacOs 10.14.6

Ace currently flags EPUB publications for missing the `schema:accessModeSufficient` metadata, while the [EPUB Accessibility 1.1](https://www.w3.org/TR/epub-a11y-11/#sec-disc-package) standard indicates that this metadata is no longer mandatory (unlike EPUB Accessibility 1.0).

Cf. #389 (same issue with `accessibilitySummary`).

**Comment by danielweck** [comment]
Related: https://github.com/daisy/ace/issues/378

This was fixed some time ago, will shortly be released in an Ace update.

**Comment by gregoriopellegrino** [comment]
Hi,
in Ace version 1.3.7 I still find the assertion:

```
{
          "@type": "earl:assertion",
          "earl:result": {
            "earl:outcome": "fail",
            "dct:description": "Publications must declare the 'schema:accessModeSufficient' metadata"
          },
          "earl:assertedBy": "Ace",
          "earl:mode": "automatic",
          "earl:test": {
            "earl:impact": "moderate",
            "dct:title": "metadata-accessmodesufficient",
            "dct:description": "Ensures a 'schema:accessModeSufficient' metadata is present",
            "help": {
              "url": "http://kb.daisy.org/publishing/docs/metadata/schema.org/index.html",
              "dct:title": "Schema.org Accessibility Metadata",
              "dct:description": "Add a 'schema:accessModeSufficient' metadata property to the Package Document"
            },
            "rulesetTags": [
              "EPUB"
            ]
          }
}
```

Can we reopen the issue? So we can start a discussion...
